### PR TITLE
[KAT-138] Improve Python algorithm documentation.

### PR DESCRIPTION
**Pull request by arthurp**

### CODE REVIEW

File: python/katana/analytics/_betweenness_centrality.pyx [hunk 1]
```diff
@@ -118,10 +115,16 @@ cdef class BetweennessCentralityPlan(Plan):
 
     @staticmethod
     def outer():
+        """
+        Parallelize outer-most iteration
```
Comment (gliga) [inline python/katana/analytics/_betweenness_centrality.pyx]: I thought we were using a single line for the comments, like """Parallelize outer-most iteration""" (and periods at the end). Minor only.

File: python/katana/analytics/_subgraph_extraction.pyx [hunk 2]
```diff
@@ -55,6 +71,12 @@ cdef class SubGraphExtractionPlan(Plan):
 
     @staticmethod
     def node_set() -> SubGraphExtractionPlan:
+        """
+        The node-set algorithm:
+
+        * Given a set of node ids, this algorithm constructs a new sub-graph
```
Comment (gliga) [inline python/katana/analytics/_subgraph_extraction.pyx]: Are those * expected here.

Also, it was a bit confusing it says "given a set of node ids", but there are no arguments

**Review by gliga — APPROVED** [review]

File: python/katana/analytics/_betweenness_centrality.pyx (hunk 1 above)
Comment (arthurp) [reply python/katana/analytics/_betweenness_centrality.pyx]: I don't like the single line form, but it's not something I push other people on. Do you think we need to be consistent about it?

Comment (gliga) [reply python/katana/analytics/_betweenness_centrality.pyx]: If we can, great.  If not, not a big deal. Go ahead with the PR no need to block on that.

Comment (arthurp) [reply python/katana/analytics/_betweenness_centrality.pyx]: Ok. I would actually prefer to consistently use the multiline form. I like how it sets out the documentation in the source, but I will look at style guides and what black does. For the moment, let's just be consistent within each file and we can create a global standard soon.

**Pull request merged**